docs/user_guide/running_studies.md

```diff
@@ -0,0 +1,520 @@
+
+In a local run, tasks are executed sequentially in your current shell and are never sent to the broker. Therefore, workers do not need to be started for a local run to execute.
+
+Local runs are typically used when designing a workflow. They help provide users a quick way to run through their study to ensure every step is executing as expected prior to doing larger production runs that should be done [in a distributed manner](#distributed-runs).
+
+To run a study locally use the `--local` option of the [`merlin run`](./command_line.md#run-merlin-run) command.
+
+```bash
+merlin run --local my_specification.yaml
+```
+
+## Distributed Runs
+
+!!! warning
+
+    Distributed runs __*require*__ a connection to both the broker and results server. See the [Configuration](./configuration/index.md) page for instructions on how to set up these connections if you haven't done so already.
+
+    If you can run `merlin info` and see no errors then your connection is working.
+
+Most of the runs done in Merlin will likely be distributed runs. These runs require:
+
+1. Tasks representing how to execute your workflow
+2. Workers to coordinate and execute these tasks
+
+In a distributed run, the [DAG](../faq.md#what-is-a-dag) created by [the `study` block](./specification.md#the-study-block) of your spec file is converted to [Celery tasks](https://docs.celeryq.dev/en/stable/userguide/tasks.html) which are then sent to queues that live on the [broker](./configuration/index.md#what-is-a-broker). To accomplish this Merlin provides the [`merlin run`](./command_line.md#run-merlin-run) command:
+
+```bash
+merlin run my_specification.yaml
+```
+
+The diagram below displays the basic flow of this process:
+
+<figure markdown>
+  ![merlin run diagram](../assets/running_studies/merlin-run-diagram.png)
+  <figcaption>High-Level Overview of "merlin run"</figcaption>
+</figure>
+
+
+Once tasks are queued on the broker, they will remain there until we initiate workers to process them.
+
+You can specify workers in [the `merlin` block](./specification.md#the-merlin-block) of your spec file. To initiate these workers on your current node, you can use the [`merlin run-workers`](./command_line.md#run-workers-merlin-run-workers) command. However, workers are typically launched in parallel which will require an allocation on your preferred HPC system prior to using the `merlin run-workers` command.
+
+=== "Launch In Parallel"
+
+    A parallel batch allocation launch is configured to run a single worker per node. This worker will then launch a number of worker processes to manage the tasks. The number of worker processes that are launched depends on the `--concurrency` value provided to the workers. By default this will be the number of CPUs on the node. The number of processes can be configured by the users (see the [Configuring Celery Workers](./celery.md#configuring-celery-workers) section for more details).
```

Comment:
It might be good to call out a warning that the total number of workers launched is  ( concurrency * number of nodes ), since concurrency is shorthand for -workers per node-

Reply:
Good call